pkg/kubectl-explain/components/SlideInPanel.vue

```diff
@@ -35,11 +35,20 @@ export default {
     };
   },
 
-  created() {
-    useWatcherBasedSetupFocusTrapWithDestroyIncluded(() => this.isOpen, '.slide-in', {
-      escapeDeactivates: false,
-      allowOutsideClick: true
-    });
+  watch: {
+    // better to trigger focus-trap based on "isOpen" rather than using the "created" hook
+    // because re-opening an already loaded resource data would not run the "created" hook
+    // and therefore not triggering the focus trap
+    isOpen(neu, old) {
+      if (neu && neu !== old) {
+        useWatcherBasedSetupFocusTrapWithDestroyIncluded(() => this.isOpen, '[data-testid="slide-in-panel-resource-explain"]', {
+          escapeDeactivates: false,
+          allowOutsideClick: true,
+          // putting the initial focus on the first element that is not conditionally displayed
+          initialFocus:      '[data-testid="slide-in-panel-close-resource-explain"]'
+        });
+      }
+    }
```

Comment:
I get the following error when I navigate to Local Cluster => Nodes => Click on Describe Resource:

```
Uncaught runtime errors:
ERROR
Your focus-trap must have at least one container with at least one tabbable node in it at all times
updateTabbableNodes@https://127.0.0.1:8005/js/chunk-vendors.js:48797:13
activate@https://127.0.0.1:8005/js/chunk-vendors.js:49251:9
./shell/composables/focusTrap.ts/useWatcherBasedSetupFocusTrapWithDestroyIncluded/</</<@https://127.0.0.1:8005/js/index.js:39228:39
```

I think that it has something to do with this change. This only happens when I first render the panel; subsequent renders will not throw the error.

Reply:
Perhaps.. The issue must've been on my end. I attempted a repro in a fresh environment and I don't see the behavior any more.